### DISCUSSION TITLE
2nd folder pane shows now files or folder

### DISCUSSION THREAD

**Discussion by artee-s2t** [WinMerge/winmerge#2356] [Q&A]
Hi @everyone,

I'm currently running 2.16.40.0 version. I'm viewing 2 panes. 
As you can see in the screenshot only the left or the first pane is showing me the folders and files. However on the right pane or "B" there is none.

Hope you can point me out to the right direction/settings on how to show the files.
I've already uninstall and reinstalled the application and it is still the same. 

By the way I'm using winmerge  on windows 10.

Looking forward for your suggestion. TIA!

![image](https://github.com/WinMerge/winmerge/assets/56582525/1c55bdf7-1659-4765-8bed-358e0bba16d9)

**Comment by sdottaka** [comment]
In the WinMerge folder comparison window, when you are comparing two folders, a single pane shows information about both folders together, rather than two panes.

Unfortunately, WinMerge does not currently support two-pane folder comparison windows.

**Comment by bradlycox** [comment]
+1 for this feature request. I came to Winmerge in the quest to replace my venerable FlashFXP with a modern equivalent, but I can't deal with the contents of the two folders merged like this. You cannot get an immediate understanding of what is in each and whether you have landed on the correct folders. When merging I want to see the folders show up in the target directory and when finished the total length of the listings in each folder side-by-side.
